cord/client.py

```diff
@@ -384,6 +386,31 @@ def get_project(self):
         """
         return self._querier.basic_getter(Project)
 
+    def add_users(self, user_emails: List[str], user_role: ProjectUserRole):
```

Comment:
Please return a data class here so that it is easy for them access the returned object